test/unit/server-config-configmap.bats

```diff
@@ -61,6 +61,66 @@ load _helpers
   [ "${actual}" = "true" ]
 }
 
+#--------------------------------------------------------------------
+# global.metrics.enabled & ui.enabled
+
+@test "server/ConfigMap: creates ui config with .global.metrics.enabled enabled & ui enabled" {
+  cd `chart_dir`
+  local actual=$(helm template \
+      -s templates/server-config-configmap.yaml  \
+      --set 'global.metrics.enabled=true' \
+      --set 'ui.enabled=true' \
+      . | tee /dev/stderr |
+      yq '.data["ui-config.json"] | length > 0' | tee /dev/stderr)
+  [ "${actual}" = "true" ]
+}
+
+@test "server/ConfigMap: creates ui config with .ui.metrics.enabled enabled & ui enabled" {
+  cd `chart_dir`
+  local actual=$(helm template \
+      -s templates/server-config-configmap.yaml  \
+      --set 'ui.metrics.enabled=true' \
+      --set 'ui.enabled=true' \
+      . | tee /dev/stderr |
+      yq '.data["ui-config.json"] | length > 0' | tee /dev/stderr)
+  [ "${actual}" = "true" ]
+}
+
+@test "server/ConfigMap: does not create ui config with .ui.metrics.enabled disabled & ui enabled" {
```

Comment:
one other super-super-super-minor nit, you're mixing `ui enabled` and `ui.metrics.enabled enabled` and the double-enabled is a bit funky to read.
Can we standardize on something like `ui.metrics enabled and ui enabled` ?